app/main.py

```diff
@@ -98,6 +101,9 @@ def grab_secrets(self):
                    _logger.info('Retrieving secret name:%s with version: %s', key_name, key_version)
            	   secret = client.get_secret(vault_base_url, key_name, key_version)
                    output_path = os.path.join(secrets_output_folder, key_name)
+                   if secret.kid is not None:
+                         _logger.info('Secret is backing certificate. Dumping private key and certificate.')
+                         self.dump_pfx(secret.value, key_name, keys_output_folder, certs_output_folder)
```

Comment:
since `dump_pfx` is a member function (and not a static one), it makes sense to pass `keys_output_folder` and `certs_output_folder` as class members using `self`

Reply:
fixed